lib/api/controllers/documentController.js

```diff
@@ -159,28 +187,28 @@ function DocumentController (kuzzle) {
     assertHasId(request);
     assertIdStartsNotUnderscore(request);
 
-    return kuzzle.validation.validationPromise(request, false)
+    return this.kuzzle.validation.validationPromise(request, false)
       .then(newRequest => {
         modifiedRequest = newRequest;
 
         request.input.args.state = 'pending';
-        kuzzle.notifier.publish(modifiedRequest);
+        this.kuzzle.notifier.publish(modifiedRequest);
 
-        return engine.createOrReplace(modifiedRequest);
+        return this.engine.createOrReplace(modifiedRequest);
       })
       .then(response => {
-        kuzzle.indexCache.add(modifiedRequest.input.resource.index, modifiedRequest.input.resource.collection);
+        this.kuzzle.indexCache.add(modifiedRequest.input.resource.index, modifiedRequest.input.resource.collection);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 116 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)